src/tox_travis/after.py

```diff
@@ -0,0 +1,150 @@
+"""Add a flag to pause and wait for all Travis jobs to complete."""
+from __future__ import print_function
+import os
+import sys
+import json
+import time
+import py
+
+from tox.config import _split_env as split_env
+try:
+    import urllib.request as urllib2
+except ImportError:
+    import urllib2  # Python 2
+
+from .utils import TRAVIS_FACTORS, parse_dict
+
+
+def travis_after_monkeypatch():
```

Comment:
Just curious - is there a reason to monkeypatch the function instead of subclassing the session class?